alvr/xtask/src/main.rs

```diff
@@ -510,6 +511,10 @@ fn clippy() {
     .unwrap();
 }
 
+fn prettier() {
+    command::run("npx -p prettier@2.2.1 prettier --write '**/*[!.min].{css,js}'").unwrap();
```

Comment:
You should not put a specific version number in the command.

Reply:
This is needed so it is consistent with the CI's version.